visidata/plugins.py

```diff
@@ -75,10 +75,10 @@ def removePluginIfExists(self, plugin):
 
     def removePlugin(self, plugin):
         vdrc = Path(options.config)
-        oldvdrc = vdrc+'.bak'
+        oldvdrc = vdrc.with_suffix('.bak')
```

Comment:
Does this result in `.visidatarc.bak` or `.bak`?

Reply:
That leaves you with `.visidatarc.bak`.

Reply:
Although... if it were possible to have a config file like `~/my.visidatarc` then it would turn it into `~/my.bak`. We could dodge that with `vdrc.with_suffix(vdrc.suffix + '.bak')` but I'm not sure if that's a good or bad idea. Thoughts?